Tests/MapboxMapsTests/Annotations/AnnotationOrchestratorImplTests.swift

```diff
@@ -50,6 +50,49 @@ final class AnnotationOrchestratorImplTests: XCTestCase {
         impl = nil
     }
 
+    func testGesturesDisabledOnInit() {
+        XCTAssertFalse(tapGestureRecognizer.isEnabled)
+        XCTAssertFalse(longPressGestureRecognizer.isEnabled)
+    }
+
+    func testGesturesEnableForAllButPointManager() {
+        let managerId = "test-manager"
+
+        let factories: [(String, LayerPosition?) -> AnnotationManagerInternal] = [
+            impl.makeCircleAnnotationManager,
+            impl.makePolygonAnnotationManager,
+            impl.makePolylineAnnotationManager,
+        ]
+
+        for factory in factories {
+            _ = factory(managerId, nil)
+        }
+
+        XCTAssertTrue(tapGestureRecognizer.isEnabled)
+        XCTAssertTrue(longPressGestureRecognizer.isEnabled)
+
+        for factory in factories {
+            _ = factory(managerId, nil)
```

Comment:
```suggestion
```

This line looks useless as the line below it is going to remove the manager anyway.